src/host.ts

```diff
@@ -1,14 +1,14 @@
-import * as fs from "fs-extra";
-import * as ts from "typescript";
-import * as _ from "lodash";
+import {LanguageServiceHost as TypescriptLanguageServiceHost, IScriptSnapshot, ParsedCommandLine, ScriptSnapshot, sys, CompilerOptions, getDefaultLibFilePath} from "typescript";
```

Comment:
I agree, syntax such as `import {Foo, Bar} as Baz from "module"` would be a nice addition to the spec